cmd/envsubst/main.go

```diff
@@ -73,7 +76,11 @@ func main() {
 		file = os.Stdout
 	}
 	// Parse input string
-	result, err := envsubst.StringRestricted(data, *noUnset, *noEmpty)
+	parserMode := parse.AllErrors
+	if *failFast {
+		parserMode = parse.Quick
+	}
+	result, err := envsubst.StringRestricted(data, *noUnset, *noEmpty, parserMode)
```

Comment:
Remove the `Mode` argument from the `StringRestricted` function and initialize a parser and execute it here.